tests/test_installation.py

```diff
@@ -0,0 +1,27 @@
+from pytest_bdd import scenario, given, when, then
+from pytest_bdd import parsers
+import os
+import subprocess
+
+@scenario('installation.feature', 'Run basic installation')
+def test_installation():
+    pass
+
+@given('A complete inventory')
+def inventory_bdd(inventory):
+    return inventory
+
+@when("I run 'make shell'")
```

Comment:
Do we want to sandbox test (avoid cache hit on a local dev laptop). Avoiding cache hit will slow down test, and make internet sensitive (flakiness ?)

Reply:
Nah, relying on an existing `make shell` environment should be fine.